### ISSUE TITLE
Failure in importing the database

### ISSUE THREAD

**Issue by jingwenw** [jonpchin/gochess#99]
Dear Sir,

After cloning the repo in my Ubuntu server, I got the database importing errors when running: go run main.go:

Database gochess does not exist
Please wait while database is imported...
util.go unzip There was an error in archiving the file open ./backup/gochess.zip: no such file or directory
Error in importing template database, please check logs
database.go Dbsetup FAILED to import both databases!

It seems the database is not part of the repo? Your helps will be highly appreciated!

Thanks,
James

**Comment by jonpchin** [comment]
Yes that is correct. I have not included the database in the repo.  This project is experimental and there is no support yet to have other people run their instance on their own web server.
